app/controllers/DownloadCsvController.scala

```diff
@@ -22,7 +22,7 @@ import controllers.actions.IdentifierAction
 import helpers.CashAccountUtils
```

Comment:
Could you please fix below warnings as well? These are all for unused imports

[info] compiling 102 Scala sources and 2 Java sources to /Users/vineet.tyagi/hmrc/codebase/customs-cash-account-frontend/target/scala-2.13/classes ...
[warn] /Users/vineet.tyagi/hmrc/codebase/customs-cash-account-frontend/app/views/ErrorTemplate.scala.html:1:1: Unused import
[warn] @*
[warn] ^
[warn] /Users/vineet.tyagi/hmrc/codebase/customs-cash-account-frontend/app/views/Head.scala.html:1:1: Unused import
[warn] @*
[warn] ^
[warn] /Users/vineet.tyagi/hmrc/codebase/customs-cash-account-frontend/app/views/Layout.scala.html:1:1: Unused import
[warn] @*
[warn] ^
[warn] /Users/vineet.tyagi/hmrc/codebase/customs-cash-account-frontend/app/views/cash_account.scala.html:1:1: Unused import
[warn] @*
[warn] ^
[warn] /Users/vineet.tyagi/hmrc/codebase/customs-cash-account-frontend/app/views/cash_account_exceeded_threshold.scala.html:1:1: Unused import
[warn] @*
[warn] ^
[warn] /Users/vineet.tyagi/hmrc/codebase/customs-cash-account-frontend/app/views/cash_account_no_transactions.scala.html:1:1: Unused import
[warn] @*
[warn] ^
[warn] /Users/vineet.tyagi/hmrc/codebase/customs-cash-account-frontend/app/views/cash_account_no_transactions_with_balance.scala.html:1:1: Unused import
[warn] @*
[warn] ^
[warn] /Users/vineet.tyagi/hmrc/codebase/customs-cash-account-frontend/app/views/cash_account_not_available.scala.html:1:1: Unused import
[warn] @*
[warn] ^
[warn] /Users/vineet.tyagi/hmrc/codebase/customs-cash-account-frontend/app/views/cash_account_transactions_not_available.scala.html:1:1: Unused import
[warn] @*
[warn] ^
[warn] 9 warnings found

Reply:
Need to be investigated further to find the best way, to resolve this.